app/routes/pay.ts

```diff
@@ -20,15 +22,20 @@ export default class PayRoute extends BaseRoute {
   }
 
   async model() {
-    await this.authenticator.authenticate();
+    if (this.authenticator.isAuthenticated) {
+      return {
+        courses: await this.store.findAll('course'), // For testimonials
+        regionalDiscount: await this.store.createRecord('regional-discount').fetchCurrent(),
+      };
+    } else {
+      if (this.authenticator.currentUser && this.authenticator.currentUser.hasActiveSubscription) {
```

Comment:
This is in the `else` clause for `authenticator.isAuthenticated`, doesn't look like this condition will ever trigger then? 

Let's add a test for this - a paying user viewing `/pay` should be redirected.